src/plugins/intel_npu/src/al/src/sync_infer_request.cpp

```diff
@@ -121,7 +121,7 @@ ov::SoPtr<ov::ITensor> SyncInferRequest::get_tensor(const ov::Output<const ov::N
     OPENVINO_ASSERT(foundPort.found(), "Cannot find tensor for port ", port);
 
     if (foundPort.is_input()) {
-        return _userInputTensors.at(foundPort.idx);
+        return get_user_input(foundPort.idx);
```

Comment:
added support only for the L0 backend.

Reply:
Hmm..then why would you implement this if the L0 method overrides the inherited function and you don't intend to implement the parent one properly?

Reply:
Because this could live there without any other issue(if someone plans to add support there as well). Even until now if the tensor wasn't found we had to return an empty value. Since set_tensors is not implemented, the behavior doesn't change.

Right now, I don't plan to add support for this in the sync_infer_request. Didn't see the benefit.